src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -337,10 +375,16 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     assetPropertyService.assetPropertyNames(lang)
   }
 
+  object TierekisteriInternalServerError {
+    def apply(body: Any = Unit, headers: Map[String, String] = Map.empty, reason: String = "") =
+      ActionResult(ResponseStatus(555, reason), body, headers)

```

Comment:
I believe we have use this different status code to open a different message on client side "Tierekisteri error".